src/test/java/core/basesyntax/strategy/impl/SupplyOperationHandlerTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SupplyOperationHandlerTest {
+    private static final FruitTransaction.Operation SUPPLY = FruitTransaction.Operation.SUPPLY;
+    private static final Integer VALID_VALUE = 20;
+    private static OperationHandler operationHandler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        operationHandler = new SupplyOperationHandler();
+    }
+
+    @Before
+    public void setUp() {
+        FruitStorage.fruits.put("banana", 100);

```

Comment:
can you do this in block Before Class? or you have to do this in Before block? I don't remember exactly but I think you can try in Before class block. I think it will be more readable.

Reply:
fixed. It`s just wrong naming again(